core-bundle/src/Controller/ImagesController.php

```diff
@@ -60,6 +60,6 @@ public function __invoke(string $path): Response
             throw new NotFoundHttpException($exception->getMessage(), $exception);
         }
 
-        return new BinaryFileResponse($image->getPath());
+        return new BinaryFileResponse($image->getPath(), 200, ['Cache-Control' => 'maxage=31536000']);
```

Comment:
This should be `max-age` imho. It's `s-maxage` and `max-age` (I always wondered why this is...).

Reply:
ecfaff2b8ee70d6ee732961fec2587e2c185ea97